app/src/sys/win/command.c

```diff
@@ -27,7 +32,7 @@ cmd_execute(const char *const argv[], HANDLE *handle) {
     memset(&si, 0, sizeof(si));
     si.cb = sizeof(si);
 
-    char cmd[256];
+    char cmd[512];
```

Comment:
Oh, thank you: https://github.com/Genymobile/scrcpy/issues/1358#issuecomment-862989748